source/common/protobuf/utility.cc

```diff
@@ -1043,4 +1043,44 @@ void TimestampUtil::systemClockToTimestamp(const SystemTime system_clock_time,
           .time_since_epoch()
           .count()));
 }
+
+void StructUtil::update(ProtobufWkt::Struct& obj, ProtobufWkt::Struct const& with) {
```

Comment:
Nit: `const ProtobufWkt::Struct& with`

Reply:
Sorry for this... Fixed in 56ea00fc5efb3a221d8dc370bdf5e78263216900